src/core/public/http/http_service.test.ts

```diff
@@ -403,11 +419,49 @@ describe('interception', () => {
       },
     });
 
-    await expect(http.fetch('/my/wat')).rejects.toThrow(/HTTP Intercept Halt/);
+    http
+      .fetch('/my/path')
+      .then(() => {
+        pending = false;
+      })
+      .catch(() => {
+        pending = false;
+      });
+    await delay(1000);
+
     expect(fetchMock.called()).toBe(true);
+    expect(pending).toBe(true);
     expect(order).toEqual(['Request 3', 'Request 2', 'Request 1', 'Response 1']);
   });
 
+  it('should skip remaining interceptors when controller halts during responseError', async () => {
```

Comment:
Based on this description, I'd expect that there were multiple inteceptors registered in this test, and that only one gets called.